src/bindings/lua/wreck.lua

```diff
@@ -470,13 +475,24 @@ function wreck.logstream (arg)
     l.watchers = {}
     for i = 0, arg.nnodes - 1 do
         local key = kvs_path (f, arg.jobid)..".log."..i
-        local iow, err = f:iowatcher {
-            key = key,
-            handler = function (iow, r)
-                if not r then return end
-                io.stderr:write (r.."\n")
-            end
-        }
+        local iow
```

Comment:
> The problem with using the reactor is that log streams aren't properly
terminated with EOF at job end, so reactor based read of the kz
files doesn't know when it is done...

This is fine, but doesn't KZ_FLAGS_NOFOLLOW allow this to work with the reactor?

Reply:
Yeah, this commit was done before the KZ_FLAGS_NOFOLLOW work, so I should probably take a second look. I'd like to try and see if the reactor would indeed exit normally with a set of kz streams open with KZ_FLAGS_NOFOLLOW. The way it is done now might be more predictable, but at the very least the commit message is misleading, sorry.